classes/01class/exercises/c01-aws01/ramiray/README.md

```diff
@@ -0,0 +1,57 @@
+Task1 :
+ 
+1. Command to create security Group and adding the rules to the SG. 
+
+```
+aws ec2 create-security-group --group-name Devopsacademy-sg --description "My security group" --vpc-id vpc-7fd6d818
```

Comment:
The exercise said you could use the "default VPC", which means you did not need to specify the `--vpc-id` if that is the default VPC.

That said, I usually prefer to do like you did -- explicitly setting the VPC -- I think it is safer like you did.